logging/logging.go

```diff
@@ -31,6 +31,8 @@ const (
 
 var (
 	DefaultDirName = filepath.Join("logs", "csp")
+
+	defaultLogMsgBufferSize = 256
```

Comment:
maybe make it const is better?

Reply:
A const is not convenient for benchmark test.

Reply:
From my perspective, Benchmark is to test performance and shouldn't impact the code.

Reply:
OK, I'll refine it.

Reply:
OK, I'll refine it.